lib/helpers/is-interactive-element.js

```diff
@@ -14,23 +14,32 @@ const INTERACTIVE_TAG_NAMES = [
   'textarea',
 ];
 
-// Spec: https://www.w3.org/TR/wai-aria/complete#widget_roles
+// Spec: https://www.w3.org/TR/wai-aria-1.1/#widget_roles
 
 const ARIA_WIDGET_ROLES = [
   'button',
   'checkbox',
+  'combobox',
+  'grid',
   'link',
+  'listbox',
+  'menu',
+  'menubar',
```

Comment:
```suggestion
```
An element with this role can contain interactive elements but is not itself interactive.